pkg/api/server/register_exec.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func (s *APIServer) registerExecHandlers(r *mux.Router) error {
-	// swagger:operation POST /containers/{name}/create compat createExec
+	// swagger:operation POST /containers/{name}/exec compat createExec
```

Comment:
If name may contain a '/' then

```suggestion
	// swagger:operation POST /containers/{name:.*}/exec compat createExec
```

Reply:
Container names allows letters, numbers, and underscores only